### ISSUE TITLE
Add "we are hiring" button to the bottom right of desktop pages

### ISSUE THREAD

**Issue by larsyencken** [owid/owid-grapher#1356]
## Proposal

We have "Subscribe" and "Feedback" buttons on the bottom right for desktop users. We should add a "We are hiring!" button there too, since we are likely to have several open roles people could look at.

<img width="606" alt="Screen Shot 2022-04-01 at 12 07 12" src="https://user-images.githubusercontent.com/36012/161242925-a892387e-69c7-4b65-b3b1-d12c72660a96.png">

**Comment by larsyencken** [comment]
@mathisonian Hey! Gavrilov and I were just discussing this. It feels like the floating nav buttons here are such prime real estate that we should use them very rarely. We're on the fence as to whether a "We're hiring" button is a good payoff for this space. Care to weigh in and tip us to one side or the other? 😄

**Comment by mathisonian** [comment]
I agree we should use the floating buttons sparingly. A few questions that might help...

- Are we lacking quality candidates for the existing roles?
- Do we think that the people that are on the site and seeing these buttons are likely to represent a good pool of candidates?
- What is the click through rate on these buttons? 

If the answers to either of the first two are _no_, or the answer to the third is _very low_, I'd be hesitant to add this. But if we think it could significantly improve hiring maybe worth it? If we do add, I'd consider changing the text to "We're hiring!" or just "Jobs" so that it takes up slightly less horizontal space

**Comment by larsyencken** [comment]
The motivating roles for this were the ops officer role and the head of product & design. The former role is now closed. The latter role would be seen only very rarely by an appropriate candidate. These two provide a kind of answer for (1) and (2) above. I think this is enough reason not to do it.

Closing as "wont fix".

**Comment by larsyencken** [comment]
After some discussion in leads, reopening this as something we've decided to do and tagging it as high prio.
